Description: Unneccessary shebang line in scripts/admin.pl
 Correct issue with unneccessary shebang line in scripts/admin.pl
Bug: http://sourceforge.net/tracker/?func=detail&aid=3473848&group_id=175965&atid=875350
Author: Robert James Clay <jame@rocasa.us>

--- a/scripts/admin.pl
+++ b/scripts/admin.pl
@@ -1,4 +1,3 @@
-#!/usr/bin/perl
 package LedgerSMB::Scripts::admin;
 use strict;
 
